jwst/datamodels/schemas/slitmeta.schema.yaml

```diff
@@ -0,0 +1,98 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/fits-schema/fits-schema"
+id: "http://stsci.edu/schemas/jwst_datamodel/slitmeta.schema"
+type: object
+  properties:
+    meta:
```

Comment:
I think moving all of these slit meta properties under `meta` is a big change that will probably break lots of code.  We should eventually do it, but probably in a different PR?  I believe there's another issue for this, and if not, we should file one.